src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -263,7 +263,7 @@ function assertValidProps(component, props) {
   }
   invariant(
     props.style == null || typeof props.style === 'object',
-    'The `style` prop expects a mapping from style properties to values, ' +
+    'The `style` prop of '+component._currentElement._owner._instance.__proto__.constructor.displayName + ' expects a mapping from style properties to values, ' +
```

Comment:
Woops must have been sleepy when I submitted that. I would change it to say
'check the render method of '+code_snippet

I read the issue as being it would be helpful if the error message
contained the name of the react class that has the render method where the
style prop has bad syntax, which is what that piece of code returns
On Jul 24, 2015 11:03 PM, "Jim" <notifications@github.com> wrote:

> In src/renderers/dom/shared/ReactDOMComponent.js
> <https://github.com/facebook/react/pull/4482#discussion_r35480276>:
>
> > @@ -263,7 +263,7 @@ function assertValidProps(component, props) {
> >    }
> >    invariant(
> >      props.style == null || typeof props.style === 'object',
> > -    'The `style` prop expects a mapping from style properties to values, ' +
> > +    'The `style` prop of '+component._currentElement._owner._instance.__proto__.constructor.displayName + ' expects a mapping from style properties to values, ' +
>
> It's the style prop of the owner component? This doesn't look right to me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react/pull/4482/files#r35480276>.
>